packages/carequality-sdk/README.md

```diff
@@ -19,6 +19,21 @@ const Carequality = new Carequality({
 });
 ```
 
+## Local development
+
+### Testing
+
+In order to run the tests, you need to have the following environment variables set, usually in a `.env` file:
+
+```
+ORG_OID=... # The OID of the organization to be used in the tests.
+API_MODE=... # dev, staging, production
+MANAGEMENT_API_KEY=... # API key for the Carequality Management API
+ORG_CERTIFICATE=... # The certificate (public key) for the organization.
+ORG_PRIVATE_KEY=... # An RSA key corresponding to the specified orgCert.
+ORG_PRIVATE_KEY_PASSWORD=... # The password to decrypt the private key.
+```
+
```

Comment:
This is always very helpful, my future self thanks you

Reply:
😬 I ended up removing those b/c:
- it was simplified (no certs stuff)
- it was moved to packages/api, which is definitely not being updated for env vars (look for 1Pwd for a mostly ready to use `.env` - to be updated as part of this PR)